addons/OPT/GPS/fn_updateUnitIcon.sqf

```diff
@@ -0,0 +1,38 @@
+/**
+* Description:
+* Updates the icon for a unit. Effectively removes and re-adds it.
+* 
+* Author:
+* Senshi
+*
+* Arguments:
+* 0: <OBJECT/STRING> _unit     Unit
+*
+* Return Value:
+*
+* Server Only:
+* No
+* 
+* Global:
+* No
+* 
+* API:
+* No
+* 
+* Example:
+* [player] call FUNC(updateUnitIcon);
+* ["player"] call FUNC(updateUnitIcon);
```

Comment:
Siehe oben

Reply:
Da ich bei serverInit nicht antworten kann (kA warum), kapere ich den hier: Wäre sicher interessant, aber ist ja nicht besonders aufregend. Wenn, dann gehören so super generische Sachen aber auch in ein "opt_core"-Modul ausgelagert. Und das mache ich erst in dem Moment, wo ein zweites Modul die Funktion fordert :) .

Reply:
Kannst du deswegen nicht, weil das eig eine Antwort auf einen älteren comment weiter oben ist. Dort kannst du dann auch antworten ;)

Würde das einfach gleich machen. So ein opt_core oder opt_events braucht ihr früher oder später sowieso. Und wenn sowas schon da ist, ist die Wahrscheinlichkeit größer dass das dann auch wiederverwendet wird, anstatt ne eigene Lösung dafür zu schmieden. Außerdem ist das ja quasi kein Aufwand ;)